server/src/enumerate.rs

```diff
@@ -59,6 +59,7 @@ crate::binding_utils::gen_py_wrapper_class_for_enum!(
 crate::binding_utils::gen_py_wrapper_class_for_enum!(
     DeviceFileType,
```

Comment:
Is this still used ? Python binding are only for server which doesn't care about `DeviceFileType`

My guess is we have forgotten to remove it.

Reply:
Removed